Makefile

```diff
@@ -1,39 +1,40 @@
-COVERAGE_REPORT := coverage.txt
-COVERAGE_PROFILE := profile.out
-COVERAGE_MODE := atomic
+# Package configuration
+PROJECT = enry
+COMMANDS = cli/enry
+
+# Including ci Makefile
+MAKEFILE = Makefile.main
+CI_REPOSITORY = https://github.com/src-d/ci.git
+CI_FOLDER = .ci
+
+# If you need to build more than one dockerfile, you can do so like this:
+# DOCKERFILES = Dockerfile_filename1:repositoryname1 Dockerfile_filename2:repositoryname2 ...
+
+$(MAKEFILE):
+	@git clone --quiet $(CI_REPOSITORY) $(CI_FOLDER); \
+	cp $(CI_FOLDER)/$(MAKEFILE) .;
+
+-include $(MAKEFILE)
 
 LINGUIST_PATH = .linguist
 
 # build CLI
-VERSION := $(shell git describe --tags --abbrev=0)
-COMMIT := $(shell git rev-parse --short HEAD)
-LDFLAGS = -s -X main.Version=$(VERSION) -X main.GitHash=$(COMMIT)
+LOCAL_TAG := $(shell git describe --tags --abbrev=0)
+LOCAL_COMMIT := $(shell git rev-parse --short HEAD)
+LOCAL_BUILD := $(shell date +"%m-%d-%Y_%H_%M_%S")
+LDFLAGS = -s -X main.version=$(LOCAL_TAG) -X main.build=$(LOCAL_BUILD) -X main.commit=$(LOCAL_COMMIT)
```

Comment:
Maybe it's better just rename the variable to `LOCAL_LDFLAGS` because in the Makefile.main could be in the future a `LDFLAGS` to build the binaries for releases and it shouldn't be changed

what do you think?